.github/ISSUE_TEMPLATE/hackathon-project-template.md

```diff
@@ -1,9 +1,96 @@
 ---
 name: Brainhack Project Template
-about: Project submissions are now closed!
+about: This provides the backbone structure for your project description. Please follow along to submit your project!
 title: ''
 labels: 'Hackathon project'
 assignees: '@ohbm/project-monitors'
 
 ---
-**Project submissions for the 2020 OHBM Brainhack are now closed**
+**Guidelines**
+
+*We are very excited to meet you at the 2021 OHBM Brainhack 🎉* *To submit a project, you need to be an attendee of the 2021 OHBM Brainhack. We ask you to register first over [here](http://www.humanbrainmapping.org/HackathonReg/). Thank you!*
+
+*We have prepared a checklist to help with your project submission. Here is how to proceed:*
+ 1. *Before filling in any part, please submit this issue*
+ 2. *Check items in the checklist below as you go through them*
+ 3. *Once you are done (at least all 'required' items must be provided), please delete the "Guidelines" section add a comment saying 'hi @ohbm/project-monitors: My project is ready!'*
+
+Thank you!
+
+*After step 1 (issue submitted), we will assign a 'project monitor' to follow your submission. If at any time you need help or anything is unclear, please add a comment and ping your project monitor. Our team is here to help!*
+
+
+----------------------------
+## Project info
+<!-- *Please fill this in first and then submit the issue* -->
+
+**Title**:
+<!--Name of your awesome project. Please also update the title of the issue to be the title of your project-->
+
+**Project lead**:
+<!--Your name and GitHub login, possibly more than 1 lead-->
+
+**[Timezone](https://github.com/ohbm/hackathon2021/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#timezone)**:
```

Comment:
We need to update also the year in the "handbook"

Reply:
Do you mean [this doc,ument](https://docs.google.com/document/d/1Q9l3Ao60aGyEuXrVV7kfLEn15PchUFX062-S9tyr0Nk/edit?ts=607fe9a2)? I have done it.

Reply:
No this one: https://www.google.com/url?q=https://github.com/ohbm/hackathon2021/blob/main/.github/ISSUE_TEMPLATE/handbooks/socialmedia_role.md&sa=D&source=editors&ust=1619022438067000&usg=AOvVaw0BKPb_mvTidf1SmDfrEejv it's a sort of a wiki for contributing and volunteers

Reply:
I think all handbooks needs updating...

Reply:
I agree.